frontend-ssr/redux/exercise-submissions/exercise-submissions.actions.ts

```diff
@@ -30,3 +36,21 @@ export const updateSubmission = (id: string, newSubmission: Submission) => ({
     newSubmission,
   },
 });
+
+export const loadSubmissions = (n: number) => ({
+  type: LOAD,
```

Comment:
This action creator should already receive the new submissions as payload. In other words, all the mocking should be done beforehand, in `<ExerciseSubmissionsPage />`.